ansible/roles/cyhy_ops/tasks/main.yml

```diff
@@ -25,4 +25,6 @@
     path: /home/cyhy_ops/.ssh/authorized_keys
     line: "{{ lookup('aws_ssm', '/ssh/public_keys/{{ item }}') }}"
     create: yes
+    owner: cyhy_ops
+    group: cyhy_ops
```

Comment:
Should we explicitly set the `mode` as well for completeness?

Reply:
It couldn't hurt- done in https://github.com/cisagov/cyhy_amis/pull/275/commits/47f9887f656a4cb1da9078c40dbfd3fe452dd071.  Thanks for the suggestion.